### PR TITLE
Consultations process

### PR DESCRIPTION
**Pull request by Nishanthh**

### CODE REVIEW

File: src/app/modules/consultations/create-consultation/create-consultation.component.ts [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      }, err => {
+        this.errorService.showErrorModal(err);
+      });
+    }
+  }
+
+  test(event) {
+    console.log('event is: ', event);
+  }
+}
```
Comment (Manzurkds) [inline src/app/modules/consultations/create-consultation/create-consultation.component.ts]: @Nishanthh Please remove it

File: src/app/modules/consultations/create-consultation/create-consultation.component.ts [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    if (valid) {
+      this.step = 2;
+    }
+  }
+
+  openAddMinistryModal(event) {
+    event.preventDefault();
+    event.stopPropagation();
+    event.stopImmediatePropagation();
+    this.ministryObject.name = this.searchText;
```
Comment (Manzurkds) [inline src/app/modules/consultations/create-consultation/create-consultation.component.ts]: @Nishanthh We can remove these right?

File: src/app/modules/consultations/consultations-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { CreateConsultationComponent } from './create-consultation/create-consultation.component';
+
+const routes: Routes = [
+    { path: '', component: CreateConsultationComponent }
```
Comment (avikt) [inline src/app/modules/consultations/consultations-routing.module.ts]: Change this to- 
```suggestion
    { path: 'new', component: CreateConsultationComponent }
```

File: src/app/modules/consultations/create-consultation/create-consultation.component.html [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+          <div class="input-block">
+              <label for="url">Paste URL of consultation PDF</label>
+              <div class="input-block__border">
+                  <input type="text" placeholder="http://" name="url" #url="ngModel" id="url" [(ngModel)]="consultationInfo.url" required />
+              </div>
+              <div *ngIf="url.invalid && (url.touched || consultationForm1.submitted)" class="error-msg">
+                  Url is Required
+                </div>
+              <div class="input-block__tips">
+                  This is the URL of the PDF or any other document on the department sites
```
Comment (avikt) [inline src/app/modules/consultations/create-consultation/create-consultation.component.html]:
```suggestion
                  This is the URL of the PDF file on the government department sites
```

File: src/app/modules/consultations/create-consultation/create-consultation.component.html [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+              <div *ngIf="deadline.invalid && (deadline.touched || consultationForm1.submitted)" class="error-msg">
+                  Deadline is Required
+              </div>
+          </div>
+          <div>
+              <button type="submit" class="btn submit-btn" [ngClass]="{'active-btn' : consultationForm1.valid}">NEXT DEPARTMENT INFO  ></button>
+          </div>
+          <div class="query-block">
+              <div>Get Queries?</div>
+              <div>If you may need any kind of help with the upload of the consultation, please get in touch woth us.</div>
```
Comment (avikt) [inline src/app/modules/consultations/create-consultation/create-consultation.component.html]:
```suggestion
              <div>If you need any kind of help with creating a consultation, please get in touch with us.</div>
```

**Pull request merged**